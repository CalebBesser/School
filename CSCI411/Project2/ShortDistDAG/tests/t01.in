7 11
0 1 4
0 2 5
1 3 8
1 4 11
2 3 1
2 5 9
3 5 3
3 6 7
4 5 5
4 6 1
5 6 3
printGraph
findShortPaths 2
