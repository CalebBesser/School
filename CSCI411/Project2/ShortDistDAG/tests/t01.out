Graph is:
0: 1/4	2/5	
1: 3/8	4/11	
2: 3/1	5/9	
3: 5/3	6/7	
4: 5/5	6/1	
5: 6/3	
6: 
Shortest distances from 2 to all reachable nodes are:
2 0
3 1
5 4
6 7
